netplay/joinroom.cpp

```diff
@@ -257,13 +268,6 @@ void JoinRoom::serverChanged(QString serverName)
     webSocket->open(serverUrl);
 }
 
-void JoinRoom::connectionFailed()
```

Comment:
Why was this function removed?

Reply:
When closing the Join Room dialog, the connection failed alert would pop up. This is a bug I found in the existing code that I fixed in this PR. It seems that QWebSocket's `timeout` signal is being fired when the object is destroyed?